backend/src/email/routes/email-transactional.routes.ts

```diff
@@ -44,6 +47,11 @@ export const InitEmailTransactionalRoute = (
         .options({ convert: true })
         .lowercase(),
       attachments: Joi.array().items(Joi.object().keys().required()),
+      classification: Joi.string()
+        .uppercase()
+        .valid(...Object.values(TransactionalEmailClassification))
+        .optional(),
+      tag: Joi.string().max(255).optional(),
```

Comment:
strictly speaking I think the `.optional()` flag is itself [optional by default](https://joi.dev/api/?v=17.8.3#:~:text=Values%20(or%20keys%20in%20case%20of%20objects)%20are%20optional%20by%20default), but I think it's good to include to make it clearer